core/app/services/spree/locales/set_fallback_locale_for_store.rb

```diff
@@ -0,0 +1,17 @@
+module Spree
+  module Locales
+    class SetFallbackLocaleForStore
+      def call(store:)
+        store_default_locale = store.default_locale
+        fallbacks = store.supported_locales_list.each_with_object({}) do |locale, object|
+          object[locale] = [store_default_locale]
```

Comment:
What if there's only one locale? Will it still try to fallback using itself? This would obviously be redundant.

Reply:
It won't change anything then. I18n::Locale::Fallbacks consolidates duplicate entries in the list of fallbacks, and the first "fallback" on the list for each locale is the actual locale. Only the later entries denote actual fallbacks in case the requested one is not present.